src/main/java/vecrosen/Event.java

```diff
@@ -0,0 +1,72 @@
+package vecrosen;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Task with 2 additional arguments: a begin and end time/date.
+ */
+public class Event extends Task {
+    private final String beginString;
+    private final String endString;
+    private boolean isDateBegin;
+    private boolean isDateEnd;
+    private LocalDate beginDate;
+    private LocalDate endDate;
+
+    /**
+     * Initializes a new Event object. Starts marked as incomplete.
+     * @param description The description of the event
+     * @param begin The start date/time of the event
+     * @param end The end date/time of the event
+     */
```

Comment:
Perhaps there should be an empty line between the description and parameter section? I noticed the same issue in several other places too!